### DISCUSSION TITLE
Sub navigation

### DISCUSSION THREAD

**Discussion by devansXD** [UKHomeOffice/design-system#508] [Ideas]
This is a proposal for a new sub navigation component. On the COP Orange team we used this MOJ version of the component in the TMS prototype https://design-patterns.service.justice.gov.uk/components/sub-navigation/

![image](https://user-images.githubusercontent.com/34478190/232828761-6d2f7017-c92a-4310-ab6b-cc843c6a6a0b.png)

For our product, this enables users to access different areas of information within a single record (which is called task view).

This component tested well in the context of our product area with National Command and Regional Command users (approx. 20 altogether), as most users understood how to navigate to each section and what information would be there. Full findings are available if needed.

**Comment by benjystanton** [comment]
Hi Dave, thanks for raising this. 

For info, I'm working on the service Dave mentioned, so I'm happy to help with future questions if anything arises as this discussion progresses.
